src/sagemaker/local_session.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import datetime
+import logging
+import time
+
+import urllib3
+from botocore.exceptions import ClientError
+
+from sagemaker.image import SageMakerContainer
+from sagemaker.session import Session
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+
+class LocalSagemakerClient(object):
```

Comment:
Add api docs. All public members should have API docs, won't repeat this per member.